functions/Get-DbaKbUpdate.ps1

```diff
@@ -93,9 +110,9 @@ function Get-DbaKbUpdate {
         }
 
         # Initialize
-        if (-not $script:websession) {
-            $null = Invoke-KbTlsWebRequest -Uri "https://www.catalog.update.microsoft.com/"
-        }
+        $script:websession = $null   # Reset request header
```

Comment:
Sorry for late reply.

> This seems to reset the web session every time.

Yes, I added that line to avoid fetching information in unintended language by using cached accept-language.

I've tested fetching information in several languages in a row at the 1st screenshot. (Japanese, Deutsch)  
And the download link for a specific language seems to be fetched correctly.

Reply:
I ended up adding this, that way it doesn't reset it each time unless required. If I recall, resetting the websession slowed down the command by a bit.

```
if ($script:websession -and $script:websession.Headers."Accept-Language" -eq $Language) {
```

Reply:
confirming it works 

![image](https://user-images.githubusercontent.com/8278033/182158749-966f04c4-d881-441c-aebf-2607499d289e.png)

Reply:
> If I recall, resetting the websession slowed down the command by a bit.

I forget to consider that point. Thank you [potatoqualitee](https://github.com/potatoqualitee)!